app/models/load_balancer_listener.rb

```diff
@@ -15,4 +15,28 @@ class LoadBalancerListener < ApplicationRecord
   has_many :resource_groups, -> { distinct }, :through => :load_balancer_pool_members
 
   virtual_total :total_vms, :vms, :uses => :vms
+
+  def load_balancer_port_range_str
+    range_to_port_range_str(load_balancer_port_range)
+  end
+
+  def instance_port_range_str
+    range_to_port_range_str(instance_port_range)
+  end
+
+  private
+
+  # Convert a Range object into the string representation usually seen for port
+  # ranges (specifically, "80" or "3200-3233")
+  def range_to_port_range_str(r)
+    if r.empty?
+      ""
+    elsif r.size == 1
+      r.first.to_s
+    else
+      # Range#last apparently ignores exclusiveness without an explicit
+      # parameter
+      "#{r.first}-#{r.last(1)[0]}"
```

Comment:
Ah I see, it's a ruby problem.
The issue is that:
`(1...10).last == (1..10).last`
is `true`